web/Views/Shared/_Layout.cshtml

```diff
@@ -77,6 +77,15 @@
         </div>
     </nav>
 </header>
+@{
+    if (TempData["FlashMessage"] != null)
+    {
+        //uses warning as the default alert colour if there's no value in TempData["FlashType"]
+        <div class="alert alert-@(TempData["FlashType"] ?? "warning")" role="alert">
+                @TempData["FlashMessage"]
+        </div>
+    }
+}

```

Comment:
Could we move this further down somewhere? Maybe in the container or in the `<main>`? I'm not sure I like how it looks when it stretches across the screen 😅

Reply:
Yeah that's a good shout! I'll see if i can put it somewhere slightly less big